pages/api/cancel.ts

```diff
@@ -89,11 +94,11 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
   const t = await getTranslation(req.body.language ?? "en", "common");
 
   const evt: CalendarEvent = {
-    type: bookingToDelete?.title,
     title: bookingToDelete?.title,
+    type: bookingToDelete?.eventType?.title as string,
     description: bookingToDelete?.description || "",
-    startTime: bookingToDelete?.startTime.toString(),
-    endTime: bookingToDelete?.endTime.toString(),
+    startTime: bookingToDelete?.startTime.toISOString(),
+    endTime: bookingToDelete?.endTime.toISOString(),
```

Comment:
toISOString returns something of the format as :
`2022-01-21T09:00:00.000Z` and that isn't the desired outcome either. 
toString returns the following in my local test:
`Fri Jan 21 2022 14:00:00 GMT+0530 (India Standard Time)`
Which is the desired result. Hence, I'd suggest to explore this further and figure out why you're getting incorrect return for `startTime` and `endTime` on Cancel booking, as this is not the right solution yet, unless we standardize toISOString everywhere

Reply:
Makes sense 💡 

By testing i figured, the reschedule and create and reschedule webhook, where in the `2022-01-21T09:00:00.000Z` (iso) format.
Now, knowing that create and reschedule, are the same place, the exception could also be in the creation rescheduling route. 

If you wish  `toString` (`Fri Jan 21 2022 14:00:00 GMT+0530 (India Standard Time)`) "everywhere", I can change https://github.com/calendso/calendso/blob/main/pages/api/book/event.ts#L298-L299 to match the format:
```
    startTime: new Date(reqBody.start).toString(),
    endTime: new Date(reqBody.end).toString(),
```

Do you think this change is quite safe?

Reply:
Hmm... Nice catch... In that case, I think toISOString change at `cancel` should be fine here. The different dates mentioned in the issue report is flipped (i know right?) which is why I was confused as to what's going on. I think toISOString at cancel webhook is actually the right way to go, sorry about the confusion

Reply:
Sorry if i confused you. Just to clarify, my goal is to have the same format for all webhooks:

Before PR:
```
  "triggerEvent": "BOOKING_CREATED", // or "BOOKING_RESCHEDULED"
  "payload": {
    "startTime": "2022-01-28T00:00:00+01:00",
    "endTime": "2022-01-28T00:15:00+01:00",
```
```
  "triggerEvent": "BOOKING_CANCELLED",
  "payload": {
    "startTime": "Thu Jan 27 2022 23:00:00 GMT+0000 (Coordinated Universal Time)",
    "endTime": "Thu Jan 27 2022 23:15:00 GMT+0000 (Coordinated Universal Time)",
```
With this PR:
```
  "triggerEvent": "BOOKING_CREATED", // or "BOOKING_RESCHEDULED"
  "payload": {
    "startTime": "2022-01-28T00:00:00+01:00",
    "endTime": "2022-01-28T00:15:00+01:00",
```
```
  "triggerEvent": "BOOKING_CANCELLED",
  "payload": {
    "startTime": "2022-01-28T00:00:00+01:00",
    "endTime": "2022-01-28T00:15:00+01:00",
```

---

But I think `bookingToDelete?.endTime.toISOString()` is indeed not right here, because it would remove the offset. Instead `dayjs(bookingToDelete?.endTime).format()` will format it correctly:

```
dayjs().format()         // "2022-01-21T12:22:16+01:00"
new Date().toISOString() // "2022-01-21T12:22:16.889Z"
```

Sorry for possible further confusion 🙈
I will change it, so it will use the `dayjs` - `format`ters format, like so:

```suggestion
    startTime: bookingToDelete?.startTime ? dayjs(bookingToDelete.startTime).format() : undefined,
    endTime: bookingToDelete?.endTime ? dayjs(bookingToDelete.endTime).format() : undefined,
```

Reply:
Sounds good!

Reply:
Once these changes are set, it should be good to go!